docs/user/VAGRANT.md

```diff
@@ -35,7 +35,7 @@ Log into the Hootenanny virtual machine:
 
 # Using Hootenanny
 
-To access the iD Editor web user interface: [local Chrome browser](http://localhost:8888/hootenanny-id).
+To access the Hoot2x iD Editor web user interface: [local Chrome browser](http://localhost:8888/hootenanny-id/).
```

Comment:
Do we want to carry forward the Hoot2x nomenclature?  Since it will be the primary UI, perhaps we drop the 2x nickname.